application/src/pages/Cart/CartForm/CartFormHeader/CartFormHeader.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { CartFormHeader } from './CartFormHeader';
+import i18n from '../../../../utils/i18next';
+
+function setup(orderNum: string) {
+  const { getByText } = render(
```

Comment:
use camelCase to name functions and do not use render method in your custom function.

Reply:
fixed